netlify.toml

```diff
@@ -1,15 +1,15 @@
 [build]
 command = "yarn build"
-environment = { NODE_VERSION = "12.18.4" }
+environment = {}
```

Comment:
Can you just delete these environment object entirely?

Reply:
Ugh, yeah this is literally the same mistake I made in waterfront access and then copied over

Reply:
As I'll be in here anyway, what would you think of refactoring it in the style of [this Zola POC for netlify](https://github.com/NYCPlanning/labs-zola/pull/1132/files)

Reply:
I'd rather we didn't. I need to think about how we want that to look a little more so for now let's keep the scope of the PR as small as possible.